src/server/pps/cmds/cmds.go

```diff
@@ -1659,7 +1665,7 @@ func Cmds(mainCtx context.Context, pachCtx *config.Context, pachctlCfg *pachctl.
 	}
 	inspectDefaults.Flags().BoolVar(&cluster, "cluster", false, "Inspect cluster defaults.")
 	inspectDefaults.Flags().StringVar(&project, "project", project, "Inspect project defaults.")
-	commands = append(commands, cmdutil.CreateAliases(inspectDefaults, "inspect defaults"))
+	commands = append(commands, cmdutil.CreateAliases(inspectDefaults, "inspect defaults", "default"))
```

Comment:
nit: in other sites where we use alias synonyms. We have the singular form in the command string and plurals in the synonyms list. Could you modify to that for consistency?

Reply:
I had considered that, but defaults are always referred to as plurals, which is different from the other nouns. For example, `pachctl inspect repo` is the normal way of inspecting a repo (but `pachctl inspect repos` is allowed), but `pachctl create defaults --cluster` is the normal way of inspecting cluster defaults (but `pachctl inspect default --cluster` is allowed). @maguenza, do you have any preference here?

Reply:
The way it is now:

```
$ pachctl help default
Defaults provide default values for certain requests, e.g. when creating a pipeline.

Associated Commands:
  pachctl create defaults   Set cluster defaults.
  pachctl delete defaults   Delete defaults.
  pachctl inspect defaults  Return defaults.
  pachctl update defaults   Update defaults.
```

With the swap:

```
$ pachctl help default
Defaults provide default values for certain requests, e.g. when creating a pipeline.

Associated Commands:
  pachctl create default   Set cluster defaults.
  pachctl delete default   Delete defaults.
  pachctl inspect default  Return defaults.
  pachctl update default   Update defaults.
```

Reply:
@acohen4 @robert-uhl - after thinking about it and discussing with customer eng, we think it makes sense to keep it as `defaults` and the alias be `default`. The reason being it's grammatically correct and more intuitive for users. I think doing that way would be a better UX.